docs/man/containerd-config.toml.5.md

```diff
@@ -13,15 +14,27 @@ does not exist at the appropriate location or is not provided via the
 **--config** option containerd uses its default configuration settings, which
 can be displayed with the **containerd config(1)** command.
 
-## DESCRIPTION
+## Description
 
-The TOML file used to configure the containerd daemon settings has a short
+The **config.toml** file used to configure the containerd daemon settings has a short
 list of global settings followed by a series of sections for specific areas
 of daemon configuration. There is also a section for **plugins** that allows
 each containerd plugin to have an area for plugin-specific configuration and
 settings.
 
-## FORMAT
+## Config versions
+The content of `/etc/containerd/config.toml` must start with a version header, for example:
+```toml
+version = 3
+```
+
+The config version 3 was introduced in containerd v2.0.
+The config version 2 used in containerd 1.x is still supported and automatically
+converted to the config version 3.
+
+For the further information, see [`../PLUGINS.md`](../PLUGINS.md).
+
+## Format
```

Comment:
*** there are a few sections in this doc that talk about CRI .. it would be good, in each of those sections, to point to the cri/config.md generally and also where appropriate to point to ./containerd-cri-config2x.md  and the 1x detail to address the referenced open issue..

Reply:
Mentioned this in `synopsis`